kernel/api/src/main/java/org/sakaiproject/util/api/ValueEncryptionService.java

```diff
@@ -0,0 +1,21 @@
+package org.sakaiproject.util.api;
+
+public interface ValueEncryptionService {
```

Comment:
I don't think this is valuable outside of the our specific implementation, there are just too many different options that a caller might want. Different areas may want different keys or different encryption ciphers.